assets/ion-schema-widget.js

```diff
@@ -33,9 +30,18 @@ function loadPage() {
         value: valueInputValue,
     });
     document.getElementById("schema_type").value = schemaTypeInputValue;
+
+
+    // clear any previous validation results
+    const pre = document.getElementById('result');
+    const resultDiv = document.getElementById('resultdiv');
+    const violation = document.getElementById('violation');
+    pre.textContent = "";
+    violation.textContent = "";
+    _set_output_style(resultDiv, "primary")
 }
 
-loadPage();
+loadPage("type::{\n  name: short_string,\n  type: string,\n  codepoint_length: range::[1, 10],\n}", "\"Hello World!\"", "short_string");
```

Comment:
_Optional_: Would like to see this line changed to make the parameters stand out more from each other. The inline Ion drowns the line. Constants? Different lines for each parameter? 

Something of the sort. 

EDIT: I see this is what you do in the cases below. Would be good here too.

Reply:
sure thing